src/physics/utils/ppgrid.F90

```diff
@@ -0,0 +1,24 @@
+
+module ppgrid
+
+!-----------------------------------------------------------------------
+!
+! Purpose:  USED ONLY FOR BACKWARDS COMPATIBILITY WITH CAM!!!!
+!           PLEASE DELETE ONCE NO LONGER NEEDED BY "to_be_ccppized"
+!           PHYSICS SCHEMES!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+!           Thanks!
+!-----------------------------------------------------------------------
+
+  use physics_grid,     only: pcols => columns_on_task
+  use vert_coord,       only: pver => pver, pverp => pverp !WILL NEED TO CHANGE THESE NAMES TO
+                                                           !SOMETHING ELSE IN CAM-SIMA!
```

Comment:
i'm guessing this is a question for @nusbaume - do these variable names need to change?